src/views/v2/rsvp/attendees/attendee/name.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * Block: RSVP
+ * Attendees - Attendee Name
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/rsvp/attendees/attendee/name.php
+ *
+ * See more documentation about our Blocks Editor templating system.
+ *
+ * @link https://evnt.is/1amp Help article for RSVP & Ticket template files.
+ *
+ * @var Tribe__Tickets__Ticket_Object $rsvp The rsvp ticket object.
+ * @var string|null $step The step the views are on.
+ * @var array $attendees List of attendees for the given order.
+ * @var int $attendee_id The attendee ID.
+ *
+ * @since TBD
+ *
+ * @version TBD
+ */
+
+if ( empty( $attendees ) || empty( $attendee_id ) ) {
+	return;
+}
+
+$attendee_name = get_post_meta( $attendee_id, tribe( Tribe__Tickets__RSVP::class )->full_name, true );
+if ( empty( $attendee_name ) ) {
+	return;
+}
+
+?>
+<div class="tribe-common-b1 tribe-common-b1--bold">
```

Comment:
We should add an element-specific class, otherwise, it won't be easy to customize CSS using these generic classes.

Reply:
I figured if we ever needed to customize this specific element, we could add the class(es) at that time.

Reply:
I am referring to the customers. Suppose, you are an admin of the website and you don't like how the style is shown and want to use some CSS to alter that. It will be harder for them to apply any CSS styles without any element specific classes.

Reply:
Yes to what Rafsun shared. We add classes (following the [BEM convention](https://en.bem.info/methodology/quick-start/)) to all the elements. That way customers can customize every single element if they'd like to. That's also a bit why I'm normally trying to find names for elements that make sense and they represent what the actual thing is. My comment about using the design system was to include the classes we already have to style things out.

In our templates, we include classes in all elements, and if there's something that we can leverage from the [design system](https://github.com/the-events-calendar/tribe-common/tree/master/src/resources/postcss/base), we do it. That way, we have a centralized place where we manage styles from the design system, and we can leverage all the styles for it, and also for maintenance purposes we have everything in one place.